packages/clerk-js/src/ui/hooks/usePasswordComplexity.ts

```diff
@@ -68,7 +69,7 @@ export const usePasswordComplexity = (config: UsePasswordComplexityConfig, callb
 
   const [password, _setPassword] = useState('');
   const [failedValidations, setFailedValidations] = useState<ComplexityErrorMessages>({});
-  const { t } = useLocalizations();
+  const { t, locale } = useLocalizations();
```

Comment:
🔧  The `useLocalizations` hook could return a boolean `canUseListFormat` value that uses the utility internally (?) Just a suggestion.